frontend/src/__tests__/cypress/cypress/pages/permissions.ts

```diff
@@ -0,0 +1,69 @@
+import { TableRow } from './components/table';
+
+class UserTableRow extends TableRow {}
+
+class GroupTableRow extends TableRow {}
+
+class PermissionsTab {
+  visit(projectName: string) {
+    cy.visitWithLogin(`/projects/${projectName}?section=permissions`);
+    this.wait();
+  }
+
+  private wait() {
+    cy.findByTestId('app-page-title');
+    cy.testA11y();
+  }
+
+  findAddButton(id: string) {
```

Comment:
Can we have separate add button for both user and group instead of  passing the id as prop?